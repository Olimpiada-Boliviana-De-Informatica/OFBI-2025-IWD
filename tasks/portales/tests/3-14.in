100
480361232 482090355 93940572 820208277 571719817 437813742 92496896 945962041 666911138 859136820 912363241 128585568 230379597 471104517 625308392 35110265 11877468 523556256 697625844 366639286 789443081 553528907 33551014 620095631 415196270 581373903 266937534 38983743 398345230 640276473 591849884 332558149 537861963 432387867 171900871 50050021 650069229 598659720 480848490 289232805 963402913 982440731 402057682 457542096 610828151 9042865 502839281 423705590 970843748 730455466 498704094 24649322 725106878 348131692 201632595 222775491 962206915 470998240 89574702 6065243 173534738 701067763 923799536 116443859 105320495 385382167 138263361 948157069 29893380 505846719 451815274 934389041 314818907 37635302 171771353 16312873 393702757 558195081 263602081 240768804 880199990 891072296 88574455 479799446 387660614 215747145 730198710 825528375 532438257 725874933 651818716 780679552 199040248 437059897 286473483 828419121 393853438 918447015 26283529 662240462