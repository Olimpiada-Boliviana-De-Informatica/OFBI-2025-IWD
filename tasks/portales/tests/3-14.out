-1
480361232
-1
482090355
482090355
93940572
-1
820208277
571719817
820208277
859136820
93940572
128585568
437813742
571719817
-1
-1
482090355
666911138
230379597
697625844
523556256
11877468
571719817
366639286
571719817
230379597
35110265
366639286
625308392
581373903
266937534
523556256
415196270
128585568
38983743
640276473
591849884
480361232
266937534
945962041
963402913
398345230
437813742
598659720
-1
482090355
415196270
963402913
697625844
482090355
11877468
697625844
332558149
171900871
201632595
945962041
457542096
50050021
-1
171900871
697625844
912363241
93940572
93940572
366639286
128585568
945962041
24649322
502839281
437813742
923799536
289232805
35110265
138263361
11877468
385382167
553528907
230379597
230379597
859136820
880199990
50050021
471104517
385382167
201632595
725106878
820208277
523556256
725106878
650069229
730455466
173534738
432387867
266937534
825528375
393702757
912363241
24649322
651818716
